cli/cmd/setup.go

```diff
@@ -40,9 +39,9 @@ func init() {
 }
 
 func runSetup(cmd *cobra.Command, args []string) {
-	cfg, err := lib.NewConfigFromFile(lib.ConfigPath)
+	cfg, err := lib.NewConfigFromFile(cfgFile)
 	if err != nil {
-		log.Infof("no config file yet. %v", err)
+		log.Info("no config file yet.")
```

Comment:
When user runs the setup command and it fails to create a configuration file; 
I wouldn't find just logging the information and carrying on enough here.
Better to log an error and log information about what default configuration the program uses with sensitive ones redacted.

Reply:
fixed in updated push